ee/query-service/license/manager.go

```diff
@@ -67,6 +67,30 @@ func StartManager(dbType string, db *sqlx.DB, useLicensesV3 bool, features ...ba
 		repo: &repo,
 	}
 
+	if useLicensesV3 {
+		// get active license from the db
+		active, err := m.repo.GetActiveLicense(context.Background())
```

Comment:
Are we sure `GetActiveLicense` will work when we haven't called `LoadActiveLicense`?

Reply:
why do we need `LoadActiveLicense` to call `GetActiveLicense` ? when the query service boots up we fetch the entire details from the sqlite DB and `LoadActiveLicense` also relies on the same `GetActiveLicense` to get the active licenses from DB to set as active in license manager and proceed with validation